docs/raw/applications/oidc.md

```diff
@@ -9,7 +9,7 @@ id
 
 - Type: `string` 
```

Comment:
Please exclude the whole file from the PR, I have a much more descriptive information waiting on a different PR.